.github/workflows/check.yml

```diff
@@ -0,0 +1,35 @@
+name: Check Docs
+
+on:
+  push:
+    branches: [v3.0]
+  pull_request: null
+
+jobs:
+  sphinx:
+    name: Sphinx reStructuredText validity
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: bash
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/cache@v4
+        with:
+          path: "~/.cache/pip"
+          key: ${{ runner.os }}-pip-sphinx-lint
```

Comment:
As cache keys are immutable in the actions/cache system, this will only cache something for the first run. After that, it will find an existing cache and will never store an updated cache with a newer sphinx-lint release in it. This would make this cache useless as soon as a new release of sphinx-lint is done after the introduction of that workflow.

Reply:
Thank you.
I was thinking through the problem of a specific key.
```shell
wget -qO- https://pypi.org/pypi/shyaml/json | jq -r '."info"."version"'
```
Finally removing caching seems to be the best choice.
Is it okay with you?

Reply:
I think leaving out caching is indeed simpler.

Reply:
Done 🍏